src/phoenix/experimental/evals/__init__.py

```diff
@@ -14,7 +14,7 @@
     "llm_eval_binary",
     "OpenAiModel",
     "PromptTemplate",
-    "HALLUCINATION_PROMPT_TEMPLATE_STR",
-    "RAG_RELEVANCY_PROMPT_TEMPLATE_STR",
+    "HALLUCINATION_PROMPT_TEMPLATE",
+    "RAG_RELEVANCY_PROMPT_TEMPLATE",
```

Comment:
I think we can shorten to `RELEVANCY_PROMPT_TEMPLATE`.

Reply:
ok